packages/core/src/styles/graphs/_pie.scss

```diff
@@ -0,0 +1,6 @@
+@import "./../vendor/@carbon/type/scss/type";
+@import "./../vendor/@carbon/type/scss/font-face/sans";
+
+text.pie-label {
```

Comment:
This may need some prefixing

Reply:
i'll contain it within the prefixed chart class